internal/pkg/template/templates/cicd/buildspec.yml

```diff
@@ -51,14 +51,25 @@ phases:
       # The tag is the build ID but we replaced the colon ':' with a dash '-'.
       # We truncate the tag (from the front) to 128 characters, the limit for Docker tags
       # (https://docs.docker.com/engine/reference/commandline/tag/)
+      # Check if the params file is empty. If so, echo error msg and exit.
       - >
         for env in $pl_envs; do
           tag=$(sed 's/:/-/g' <<<"${CODEBUILD_BUILD_ID##*:}-${env}" | rev | cut -c 1-128 | rev)
           for svc in $svcs; do
           ./copilot-linux svc package -n $svc -e $env --output-dir './infrastructure' --tag $tag;
+          [ -s ./infrastructure/$svc-$env.params.json ]
```

Comment:
What does this mean 🤔

Reply:
It checks if a file is empty or not.
I've removed it, as we can just check the exit code from the `svc package` command.